src/ic/managed/verifier/src/utils.rs

```diff
@@ -0,0 +1,41 @@
+use rs_merkle::{algorithms::Sha256, Hasher};
+use serde_json::Value;
+use verity_dp_ic::{
+    crypto::config::{Config, Environment},
+    owner,
+    remittance::state::CONFIG,
```

Comment:
This CONFIG is loaded from `remittance`, however, it seems more general purpose?

Is there a better location for CONFIG? - ie. utils

Reply:
To be moved to standalone or `ecdsa`